package.json

```diff
@@ -22,14 +22,18 @@
     "ganache-cli": "6.1.8",
     "openzeppelin-solidity": "2.1.1",
     "truffle": "5.0.1",
-    "web3": "1.0.0-beta.36"
+    "web3": "1.0.0-beta.36",
```

Comment:
Since it's relatively a new repo should we use latest stable web3 version?

Reply:
Out of scope of this ticket.